package.json

```diff
@@ -1,42 +1,43 @@
 {
-  "name": "@mparticle/web-optimizely-kit",
-  "version": "2.0.2",
-  "author": "mParticle Developers <developers@mparticle.com> (https://www.mparticle.com)",
-  "description": "mParticle integration sdk for Optimizely",
-  "main": "dist/OptimizelyKit.common.js",
-  "browser": "dist/OptimizelyKit.common.js",
-  "files": [
-    "dist/OptimizelyKit.common.js"
-  ],
-  "repository": "https://github.com/mparticle-integrations/mparticle-javascript-integration-optimizely",
-  "scripts": {
-    "testKarma": "node test/boilerplate/test-karma.js",
-    "build": "ENVIRONMENT=production rollup --config rollup.config.js",
-    "watch": "ENVIRONMENT=production rollup --config rollup.config.js -w"
-  },
-  "devDependencies": {
-    "@mparticle/web-kit-wrapper": "^1.0.4",
-    "chai": "^4.2.0",
-    "karma": "^3.1.1",
-    "karma-chai": "^0.1.0",
-    "karma-chrome-launcher": "^2.2.0",
-    "karma-edge-launcher": "^0.4.2",
-    "karma-firefox-launcher": "^1.1.0",
-    "karma-ie-launcher": "^1.0.0",
-    "karma-junit-reporter": "^1.2.0",
-    "karma-mocha": "^1.3.0",
-    "karma-safari-launcher": "^1.0.0",
-    "mocha": "^2.3.2",
-    "rollup": "^1.15.6",
-    "rollup-plugin-commonjs": "^10.0.0",
-    "rollup-plugin-node-resolve": "^5.0.3",
-    "shelljs": "^0.8.3",
-    "should": "^7.1.0",
-    "watchify": "^3.9.0"
-  },
-  "dependencies": {
-    "isobject": "^4.0.0",
-    "@mparticle/web-sdk": "^2.11.1"
-  },
-  "license": "Apache-2.0"
+    "name": "@mparticle/web-optimizely-kit",
```

Comment:
Should this be part of a separate pr?

Reply:
looks like it was being prettified.  i'm okay with it being on here